Source/WTF/wtf/cocoa/RuntimeApplicationChecksCocoa.h

```diff
@@ -108,6 +108,7 @@ enum class SDKAlignedBehavior {
     ScreenOrientationAPIEnabled,
     PopoverAttributeEnabled,
     LiveRangeSelectionEnabledForAllApps,
+    DoesNotOverrideUAFromNSUserDefult,
```

Comment:
"Defult"

Reply:
Oops, typo. Fixed in https://github.com/WebKit/WebKit/commit/1d8d75f9f783f278033bb894174663bb1fe99a6d